solr/solr-ref-guide/src/jwt-authentication-plugin.adoc

```diff
@@ -158,12 +161,18 @@ Let's comment on this config:
 <11> Configure the issuer id. Will be used for validating tokens. A token's 'iss' claim must match one of the configured issuer IDs.
 <12> Configure the audience claim. A token's 'aud' claim must match 'aud' for one of the configured issuers.
 <13> This issuer is auto configured through discovery, so 'iss' and JWK settings are not required
+<14> Provides SSL certificates to trust to support self-signed SSL in the issuers.
 
 === Using non SSL URLs
 In production environments you should always use SSL protected HTTPS connections, otherwise you open yourself up to attacks.
 However, in development, it may be useful to use regular http urls, and bypass the
 security check that Solr performs. To support this you can set the environment variable `solr.auth.jwt.allowOutboundHttp=true`.
 
+=== Trusting the IdP server
+All communication with issuer server (IdP) is done over HTTPS. If the issuer uses a self signed SSL certificate, the connection will fail since the plugin enforces verification of SSL certificates on outbound traffic. By default, the root certificates present in Java's TrustStore (default or custom provided) are consulted. If you do not want to place the IdP certificates in Java's TrustStore, you can configure this plugin with either `trustedCertsFile` or `trustedCerts` options to provide a set of certificates to use when talking to IdP. This has the extra benefit of also working even if Solr is not started in SSL mode.
+
+Please configure either `trustedCerts` or `trustedCertsFile`, not both.
```

Comment:
Stronger language here, indicating that it will fail if both are configured.

Reply:
Done.